src/main/java/org/jitsi/jicofo/BridgeSelector.java

```diff
@@ -91,7 +92,8 @@
     /**
      * The map of bridge JID to <tt>BridgeState</tt>.
      */
-    private final Map<String, BridgeState> bridges = new HashMap<>();
+    private final Map<String, BridgeState> bridges
+        = new ConcurrentHashMap<>();
```

Comment:
I'm not sure why this is being done. Baring the readability and possibly the maintenance issues of not having synchronized on private methods, it looks like bridges is not protected only in getKnownBridgesCount()? I think that having bridges as a ConcurrentHashMap given all other `synchronized` all over the place makes one question why 2 things are being done when 1 would have sufficed.